staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/webhook_converter.go

```diff
@@ -186,6 +192,7 @@ func (c *webhookConverter) Convert(in runtime.Object, toGV schema.GroupVersion)
 		// TODO: Return a webhook specific error to be able to convert it to meta.Status
 		return nil, fmt.Errorf("conversion webhook for %v failed: %v", in.GetObjectKind(), err)
 	}
+	trace.Step("Request completed")
```

Comment:
I don't think we are likely to have anything concurrent here but for conformity I think we should indicate which request completed.

Reply:
the trace instance holds the context and logs a unique trace instance with every line:

https://github.com/kubernetes/kubernetes/blob/461f439b24ee43d95f328d225d1464f948f70893/vendor/k8s.io/utils/trace/trace.go#L63-L75

Reply:
I'll capture the output of a couple traces and we can review it and see how it looks.

Reply:
Here's an example output, with the trace id correlating the "request completed" with the webhook call. I'll add examples of all the trace output to the description of this issue.

```
I0611 20:09:55.596765       1 trace.go:81] Trace[723959798]: "Call conversion webhook: custom resource definition: foos.stable.example.com, desired API version: stable.example.com/v1, object count: 1, UID: 392dc2ee-3623-4770-871e-1a8041ba5253" (started: 2019-06-11 20:09:55.59189597 +0000 UTC m=+820.005173610) (total time: 1034.819304ms):
Trace[723959798]: [4.773678ms] [4.773678ms] Request completed
```

Reply:
My confusion. I had assumed we were linking these traces to the traces for the requests (Eg. Update Pod) they are part of and not that they were independent traces. It would be nice to have these traces as part of the large request. Otherwise it is hard to identify how much delay the admission webhooks are adding to a request.

Reply:
Yep, that's a follow-up. I reviewed the code briefly and it's not trivial to do with trace given how it works today. It's also a bit more broad of a change than this; there are etcd storage layer traces that are currently independent to fix. I'll open a separate issue now.